### PR TITLE
Adjust repository to work with angularjs components.

### PR DESCRIPTION
**Pull request by wtrocki**
## Motivation

Migrate code (no need for history as original repositories will be still there and we going to heavily change our code anyway. Adjust repository to work with angularjs components.

## Progress
- [x] Migration
- [x] Unit tests compatibility
- [x] Runtime testing
- [x] Documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,8 +45,4 @@ Before publishing the typescript modules, do a manual compilation step via `npm
 
  `npm run bootstrap` - perform boostrap for all modules
 
- `npm run start` - run top level applications
-
- `npm run build` - execute the build command for all modules, compiling TypeScript sources to JavaScript
-
- `npm run cleanInstall` - perform install without executing additional scripts
```
Comment (JameelB) [inline README.md:52]: Missing test and cleanInstall descriptions

File: demo/raincatcher-demo-mobile/package.json [hunk 2]
```diff
@@ -1,18 +1,18 @@
...
   },
   "scripts": {
+    "start": "grunt",
     "test": "grunt eslint"
   },
   "keywords": [
     "wfm"
   ],
-  "author": "Brian Leathem",
+  "author": "Feedhenry Raincatcher",
```
Comment (JameelB) [inline demo/raincatcher-demo-mobile/package.json]: We've been using this as the author for our other modules and repos: `feedhenry-raincatcher@redhat.com`

File: demo/raincatcher-demo-portal/package.json [hunk 3]
```diff
@@ -14,7 +13,7 @@
   "keywords": [
     "wfm"
   ],
-  "author": "Brian Leathem",
+  "author": "Feedhenry Raincatcher",
```
Comment (JameelB) [inline demo/raincatcher-demo-portal/package.json]: We've been using this as the author for our other modules and repos: `feedhenry-raincatcher@redhat.com`

File: demo/raincatcher-demo-portal/package.json [hunk 4]
```diff
@@ -1,8 +1,7 @@
 {
-  "name": "fh-wfm-demo-portal",
-  "version": "2.8.0",
-  "description": "A WFM2 PoC using the mediator pattern",
-  "repository": "https://github.com/feedhenry-raincatcher/raincatcher-demo-portal.git",
+  "name": "@raincatcher/demo-portal",
+  "version": "1.0.0",
+  "description": "A WFM example demo applications",
```
Comment (JameelB) [inline demo/raincatcher-demo-portal/package.json]: change description to `Example demo portal application`

**Review by austincunningham — APPROVED** [review]
verified

File: README.md (hunk 1 above)
Comment (wtrocki) [reply README.md:52]: Test is above. Clean install is no longer required as we do not compile typescript in this repository.

**Pull request merged**